crosswalks/swMATH.csv

```diff
@@ -0,0 +1,75 @@
+Property,swMATH
```

Comment:
You can debug locally with https://github.com/codemeta/codemeta/blob/714e4f873b01ebf89dafbb02cca8bacb30c8a605/scripts/aggregate.py to see what's wrong

Reply:
If this seems hard, please ignore the comment. I don't want to hold things back.